### ISSUE TITLE
Frontend: search box on digital identities-overview page 

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#72]
## User story
1. As a {HR-employee}
2. I want / need { to be able to search for a digital identity by their first- or surname}
3. So that {I can quickly find the identity I am looking for}

## Acceptance criteria
* search-field at the top of the page (above the table) with a "search"-button on the right side
* before clicking on the search-field, it says "Enter name."
* after clicking search, the table only shows the rows which contain the content entered name in the field "first name" or "surname"

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
